src/components/tx-flow/flows/SignMessage/SignMessage.tsx

```diff
@@ -167,6 +171,8 @@ export type ConfirmProps = BaseProps & {
 const SignMessage = ({ message, safeAppId, requestId }: ProposeProps | ConfirmProps): ReactElement => {
   // Hooks & variables
   const { setTxFlow } = useContext(TxModalContext)
+  const { setSafeMessage: setContextSafeMessage } = useContext(SafeTxContext)
+  const { needsRiskConfirmation, isRiskConfirmed, setIsRiskIgnored } = useContext(TxSecurityContext)
```

Comment:
Should we also disable the Sign button if `needsRiskConfirmation` like we do in the other flows?

Reply:
AFAIK we do not disable it in our other flows. Instead it will highlight the checkbox when trying to submit and abort the submission. That happens here as well.

I think the idea behind is that the user should understand why the submit button is disabled.